README.md

```diff
@@ -1,104 +1,113 @@
-        - [Running Tests](#running-tests)
-        - [Development](#development)
+> **Important:** Do not install this operator directly. Only install this operator using the IBM Common Services Operator. For more information about installing this operator and other Common Services operators, see [Installer documentation](http://ibm.biz/cpcs_opinstall). If you are using this operator as part of an IBM Cloud Pak, see the documentation for that IBM Cloud Pak to learn more about how to install and use the operator service. For more information about IBM Cloud Paks, see [IBM Cloud Paks that use Common Services](http://ibm.biz/cpcs_cloudpaks).
 
-<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+You can use the ibm-management-ingress-operator to install the IBM System Management ingress service. You can use IBM System Management ingress service to expose your services of the IBM Cloud Paks and IBM Cloud Platform Common Services.
 
-# Management Ingress Operator
+For more information about the available IBM Cloud Platform Common Services, see the [IBM Knowledge Center](http://ibm.biz/cpcsdocs).
 
-## Overview
+## Supported platforms
 
-This is the operator to deploy management ingress of IBM Cloud Pak foundation..
+Red Hat OpenShift Container Platform 4.3 or newer installed on one of the following platforms:
+
+- Linux x86_64
+- Linux on Power (ppc64le)
+- Linux on IBM Z and LinuxONE
+
+## Operator versions
+
+- 3.6.0
+    - Support for OpenShift 4.3 and 4.4.
+- 3.5.0
 
 ## Prerequisites
 
-- [go][go_tool] version v1.13+.
-- [docker][docker_tool] version 17.03+
-- [kubectl][kubectl_tool] v1.13.0+
-- [operator-sdk][operator_install]
-- Access to a Kubernetes v1.13.0+ cluster
+Before you install this operator, you need to first install the operator dependencies and prerequisites:
+
+- For the list of operator dependencies, see the IBM Knowledge Center [Common Services dependencies documentation](http://ibm.biz/cpcs_opdependencies).
+- For the list of prerequisites for installing the operator, see the IBM Knowledge Center [Preparing to install services documentation](http://ibm.biz/cpcs_opinstprereq).
+
+## Documentation
+
+To install the operator with the IBM Common Services Operator follow the the installation and configuration instructions within the IBM Knowledge Center.
+
+- If you are using the operator as part of an IBM Cloud Pak, see the documentation for that IBM Cloud Pak. For a list of IBM Cloud Paks, see [IBM Cloud Paks that use Common Services](http://ibm.biz/cpcs_cloudpaks).
+- If you are using the operator with an IBM Containerized Software, see the IBM Cloud Platform Common Services Knowledge Center [Installer documentation](http://ibm.biz/cpcs_opinstall).
+
+## SecurityContextConstraints Requirements
+
+The IBM System  Management ingress service now don't support running with the OpenShift Container Platform 4.3 default restricted Security Context Constraints (SCCs).
```

Comment:
Minor edit ( avoid 'now' as it can be kind of claiming a future change)

The IBM System Management ingress service does not support running with the OpenShift Container Platform 4.3 default restricted Security Context Constraints (SCCs).

Reply:
Updated.